src/index.html

```diff
@@ -3,15 +3,162 @@
   <head>
     <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+     name="viewport"
+     content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    name="Personal information"
+    method="post"
+    autocomplete="off">
+      <fieldset class="group">
+        <legend> Personal information</legend>
+
+        <label for="surname">Surname:</label>
+        <input
+        class="input"
+        type="text"
+        id="surname"
+        name="surname">
```

Comment:
```suggestion
        <input
          class="input"
          type="text"
          id="surname"
          name="surname"
        >
```
Fix code style according to [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) p.3